processor/README.md

```diff
@@ -184,6 +228,8 @@ for more information.
 
 ## <a name="attributes"></a>Attributes Processor
 
+Supported data sources: spans
```

Comment:
I feel like this info is quite important and wonder if it could be presented more clearly somehow, e.g. perhaps convert the ToC at the top to a table that includes this info?

Reply:
Yes, I agree -- trying to get something out, but a refactor is needed. I will take a look this weekend.

Reply:
s/spans/traces? and everywhere to match the pipeline data types.